e2e-polybft/e2e/multiple_bridges_test.go

```diff
@@ -0,0 +1,471 @@
+package e2e
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/big"
+	"path"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/0xPolygon/polygon-edge/command/bridge/common"
+	bridgeHelper "github.com/0xPolygon/polygon-edge/command/bridge/helper"
+	polycfg "github.com/0xPolygon/polygon-edge/consensus/polybft/config"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi"
+	"github.com/0xPolygon/polygon-edge/crypto"
+	"github.com/0xPolygon/polygon-edge/e2e-polybft/framework"
+	helperCommon "github.com/0xPolygon/polygon-edge/helper/common"
+	"github.com/0xPolygon/polygon-edge/txrelayer"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	chainConfigFile = "genesis.json"
+)
+
+func TestE2E_Multiple_Bridges_ExternalToInternalTokenTransfer(t *testing.T) {
```

Comment:
Why all the deposits are made with the same combinations of source and destination chain ids (it is always 100 for source chain id and 1337 for the destination one)?

```go
=== RUN   TestE2E_Multiple_Bridges_ExternalToInternalTokenTransfer/bridge_ERC20_tokens
    multiple_bridges_test.go:115: [100⇄1337] Root ERC20 smart contract was successfully deployed on the external chain 1337 at address 0xd209b9FfB79483CF41073729303E0876a0bfBf0f
    multiple_bridges_test.go:115: [100⇄1337] The deposit was made for the account 0xc5283eEB9B99f0443697d36c0cefd0be21F5d724
    multiple_bridges_test.go:115: [100⇄1337] The deposit was made for the account 0x3BE95aFD2F9547051ca3B1Ec7a7c989DD7b101a4
    multiple_bridges_test.go:115: [100⇄1337] The deposit was made for the account 0x91e59BC18bA6fB3c16fdAB35Aa62c24C509bbCf7
    multiple_bridges_test.go:115: [100⇄1337] The deposit was made for the account 0xDAacADbc01D24dADe0A0A2806C0c7c958Da6744c
    multiple_bridges_test.go:115: [100⇄1337] The deposit was made for the account 0x93B31930A0cb512E6a8B9Ce1C92cFC0c113a014b
```